Jasmine/Levels/ViewControllers/GameLevelList/GameLevelListViewController.swift

```diff
@@ -30,7 +30,7 @@ class GameLevelListViewController: UIViewController {
             levelsCategoriesToggle.selectedSegmentIndex
                 = newValue ? GameLevelListViewController.builtInLevelsIndex
                            : GameLevelListViewController.customLevelsIndex
-            reloadAllData()
+            onLevelsCategoriesSwitched()
```

Comment:
you mean it doesn't work when it's changed? If it doesn't, switch back to reloadAllData() bah.